.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
Most of our sites that uses decanter v7 is on node 18 now

Reply:
If you want to enforce node 18 on the projects that are using this, you would add the engine version to `package.json`. I think it is fine not to require node 18 as I believe this package is still very much compatible with node 16. For us when working on this project, node 18 makes sense here.

Reply:
Got it - that sounds good. No enforcing it is 👍🏼 😄